solver/src/solver/paraswap_solver/api.rs

```diff
@@ -132,8 +132,10 @@ pub struct PriceQuery {
     pub to_decimals: usize,
     /// amount of source token (in the smallest denomination, e.g. for ETH - 10**18)
     pub amount: U256,
-    // Type of order
+    /// Type of order
     pub side: Side,
+    /// The list of DEXs to exclude from the computed price route.
+    pub exclude_dexs: Option<Vec<String>>,
```

Comment:
Can remove Option because empty Vec is same as None.

Reply:
I just left it in case `excludeDEXS=` was different than omitting it completely. Will change though.